### PR TITLE
Add apt config example

### PR DESCRIPTION
**Pull request by maximecolin**

### CODE REVIEW

File: custom/ansible/group_vars/app.yml.twig [hunk 1]
```diff
@@ -4,6 +4,15 @@
 
 app_patterns:
 
+  #######
+  # Apt #
+  #######
+
+  # apt_preferences:
+  #   - foobar@backports
```
Comment (nervo) [inline custom/ansible/group_vars/app.yml.twig:12]: Could you add a line separator here...

File: elao-symfony/ansible/group_vars/app.yml.twig [hunk 2]
```diff
@@ -4,6 +4,15 @@
 
 app_patterns:
 
+  #######
+  # Apt #
+  #######
+
+  # apt_preferences:
+  #   - foobar@backports
```
Comment (nervo) [inline elao-symfony/ansible/group_vars/app.yml.twig:12]: ... and there ?

**Review by nervo — CHANGES_REQUESTED** [review]

File: custom/ansible/group_vars/app.yml.twig (hunk 1 above)
Comment (maximecolin) [reply custom/ansible/group_vars/app.yml.twig:12]: Done

File: elao-symfony/ansible/group_vars/app.yml.twig (hunk 2 above)
Comment (maximecolin) [reply elao-symfony/ansible/group_vars/app.yml.twig:12]: Done

**Review by nervo — APPROVED** [review]

**Comment by nervo** [comment]
I just let @chalasr doing the merge :)

**Review by ogizanagi — APPROVED** [review]

**Review by chalasr — APPROVED** [review]

**Pull request merged**